src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -414,6 +414,94 @@ public static void AtomFeedPositiveTest()
             }
         }
 
+        [Fact]
+        public static void RssEntryPositiveTest()
+        {
+            string filePath = @"RssEntry.xml";
+            string serializeFilePath = Path.GetTempFileName();
+
+            try
+            {
+                SyndicationItem feedObjct = null;
+                using (XmlReader reader = XmlReader.Create(filePath))
+                {
+                    Rss20ItemFormatter rss20ItemFormatter = new Rss20ItemFormatter();
+                    rss20ItemFormatter.ReadFrom(reader);
+                    feedObjct = rss20ItemFormatter.Item;
+                    reader.Close();
```

Comment:
not needed. The Dispose would close the reader.

Reply:
Done.